ktor-server/ktor-server-plugins/ktor-server-websockets/jvm/src/io/ktor/server/websocket/WebSocketServerSession.kt

```diff
@@ -29,6 +32,68 @@ public interface DefaultWebSocketServerSession : DefaultWebSocketSession, WebSoc
  */
 public val WebSocketServerSession.application: Application get() = call.application
 
+/**
+ * Converter for web socket session
+ */
+public val WebSocketServerSession.converter: WebsocketContentConverter? get() =
```

Comment:
I can't find a similar one for client. Am I missing something here?

Reply:
I've forgot about it by chance. I've added now